com.jonathankang.Weibird.json

```diff
@@ -0,0 +1,69 @@
+{
+    "app-id" : "com.jonathankang.Weibird",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "master",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "weibird",
+    "desktop-file-name-prefix" : "(Master)",
+    "finish-args" : [
+        "--device=dri",
+        "--share=ipc",
+        "--share=network",
+        "--socket=wayland",
+        "--socket=x11",
+        "--filesystem=host",
+        "--talk-name=ca.desrt.dconf",
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+    ],
+    "build-options" : {
+        "cflags" : "-O0 -g"
+    },
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig"
+    ],
+    "modules" : [
+        {
+            "name" : "meson",
+            "buildsystem" : "simple",
+            "build-commands" : [
+                "python3 setup.py install --prefix=/app"
+            ],
+            "cleanup" : ["*"],
+            "sources" : [
+                {
+                    "type" : "archive",
+                    "url" : "https://files.pythonhosted.org/packages/df/df/8d5c006f87a3c75f588aeadcf089726bba1e567997a1088655fe7ed68f3d/meson-0.48.0.tar.gz",
+                    "sha256" : "ecc6b319d86362d1ae2a64c40738f15dff5921b01cdba5d6506c877e4dd55c61"
+                }
+            ]
+        },
+        {
+            "name" : "librest",
+            "buildsystem" : "autotools",
```

Comment:
autotools is the default build system (but it's a nitpick really)

Reply:
Sure. Removed.